src/com/pentalog/pentastagiu/homework2/shop/Product.java

```diff
@@ -0,0 +1,23 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+public class Product {
+    protected int id;
+    protected String name;
+    protected int price;
+
+    public Product() {
```

Comment:
oare are rost sa lasi constructorul asta? are rost sa ai produse in magazinul tau care nu au id/nume/pret setate?